packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.test.tsx

```diff
@@ -56,4 +58,202 @@ describe('GroupedList', () => {
+
+    wrapper.unmount();
+  });
+
+  it('renders no rows if group is collapsed', () => {
+    const _selection = new Selection();
+    const _items: Array<{ key: string }> = [{ key: '1' }, { key: '2' }, { key: '3' }];
+    const _groups: Array<IGroup> = [
+      {
+        count: 3,
+        hasMoreData: true,
+        isCollapsed: true,
+        key: 'group0',
+        name: 'group 0',
+        startIndex: 0,
+        level: 0,
+        children: []
+      }
+    ];
+
+    function _onRenderCell(nestingDepth: number, item: any, itemIndex: number): JSX.Element {
+      return (
+        <DetailsRow
+          columns={Object.keys(item)
+            .slice(0, 2)
+            .map(
+              (value): IColumn => {
+                return {
+                  key: value,
+                  name: value,
+                  fieldName: value,
+                  minWidth: 300
+                };
+              }
+            )}
+          groupNestingDepth={nestingDepth}
+          item={item}
+          itemIndex={itemIndex}
+          selection={_selection}
+          selectionMode={SelectionMode.multiple}
+        />
+      );
+    }
+
+    const wrapper = mount(<GroupedList items={_items} groups={_groups} onRenderCell={_onRenderCell} selection={_selection} />);
+
+    const listRows = wrapper.find(DetailsRow);
+    expect(listRows.length).toBe(0);
+
+    wrapper.unmount();
+  });
+
+  it('renders the specified count of rows if "Show All" is to be displayed and all rows once "Show All" is clicked', () => {
+    const _selection = new Selection();
+    const _items: Array<{ key: string }> = [{ key: '1' }, { key: '2' }, { key: '3' }];
+    const _groups: Array<IGroup> = [
+      {
+        count: 1,
+        hasMoreData: true,
+        isCollapsed: false,
+        key: 'group0',
+        name: 'group 0',
+        startIndex: 0,
+        level: 0
+      }
+    ];
+
+    function _onRenderCell(nestingDepth: number, item: any, itemIndex: number): JSX.Element {
+      return (
+        <DetailsRow
+          columns={Object.keys(item)
+            .slice(0, 2)
+            .map(
+              (value): IColumn => {
+                return {
+                  key: value,
+                  name: value,
+                  fieldName: value,
+                  minWidth: 300
+                };
+              }
+            )}
+          groupNestingDepth={nestingDepth}
+          item={item}
+          itemIndex={itemIndex}
+          selection={_selection}
+          selectionMode={SelectionMode.multiple}
+        />
+      );
+    }
+
+    const wrapper = mount(<GroupedList items={_items} groups={_groups} onRenderCell={_onRenderCell} selection={_selection} />);
+
+    let listRows = wrapper.find(DetailsRow);
+    expect(listRows.length).toBe(1);
+
+    const groupShowAllElement = wrapper.find(GroupShowAll);
+
+    groupShowAllElement.simulate('click');
+
+    listRows = wrapper.find(DetailsRow);
+    expect(listRows.length).toBe(1);
```

Comment:
woops!

Reply:
Fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/9610/commits/5d6d4853ece1cc5526330eb30d73c9b9dabebc93!